### PR TITLE
Font AvenirNext, router transition, glow styling and glow dots 

### PR DESCRIPTION
**Pull request by BartoszBilejczyk**

### CODE REVIEW

File: src/app/auth/auth.component.ts [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
   selector: 'app-auth',
   templateUrl: './auth.component.html',
-  styleUrls: ['./auth.component.scss']
+  styleUrls: ['./auth.component.scss'],
+  animations: [fadeAnimation]
 })
-export class AuthComponent implements OnInit {
 
+export class AuthComponent implements OnInit {
+  testState = 'normal';
```
Comment (aqum) [inline src/app/auth/auth.component.ts]: I don't see this one being used anywhere.

File: src/app/animations/fade.animation.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      ),
+      query(':enter',
+        [
+          style({ opacity: 0 }),
+          animate('0.2s', style({ opacity: 1 }))
+        ],
+        { optional: true }
+      )
+    ])
+  ]);
```
Comment (KonradFak) [inline src/app/animations/fade.animation.ts]: I think this animation folder should be moved to common directory

File: src/app/auth/auth.component.ts (hunk 1 above)
Comment (BartoszBilejczyk) [reply src/app/auth/auth.component.ts]: @aqum you have good sight then! 🗡  thanks

File: src/app/app.module.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
```
Comment (KonradFak) [inline src/app/app.module.ts]: This could be in `common.module` when you move animations there

**Pull request merged**